.prettierignore

```diff
@@ -1 +1,2 @@
+types/ts-toolbelt/test/*
```

Comment:
Added this because prettier is unable to format the code properly. Guess it's the crazy amounts of type parameters that cause that.... Otherwise, everything is linted by tslint.